mmv1/third_party/terraform/services/sql/resource_sql_database_instance.go.tmpl

```diff
@@ -1212,6 +1212,7 @@ API (for read pools, effective_availability_type may differ from availability_ty
 						"psa_write_endpoint": {
 							Type:        schema.TypeString,
 							Optional:    true,
+							Computed:    true,
```

Comment:
What is the reason to add `Computed: true`? Will API return a value if `psa_write_endpoint` is not set in the API request?